components/weather_data/WeatherStationList.tsx

```diff
@@ -34,27 +35,15 @@ export const WeatherStationList: React.FC<Props> = ({center_id, requestedTime})
     return <NotFound terminal what={[new NotFoundError('no token for stations', 'weather stations')]} />;
   }
 
-  return <StationList center_id={center_id} token={metadata.widget_config.stations?.token} requestedTime={requestedTime} />;
-};
-
-const StationList: React.FunctionComponent<{
-  center_id: AvalancheCenterID;
-  token: string;
-  requestedTime: RequestedTimeString;
-}> = ({center_id, token, requestedTime}) => {
-  const mapLayerResult = useMapLayer(center_id);
-  const mapLayer = mapLayerResult.data;
-  const weatherStationsResult = useWeatherStationsMetadata(center_id, token);
-  const weatherStations = weatherStationsResult.data;
-
-  if (incompleteQueryState(mapLayerResult, weatherStationsResult) || !mapLayer || !weatherStations) {
-    return <QueryState results={[mapLayerResult, weatherStationsResult]} />;
-  }
-
   if (center_id === 'NWAC') {
-    return <NWACStationList center={center_id} mapLayer={mapLayer} stations={weatherStations} requestedTime={requestedTime} />;
+    return <NWACStationList token={metadata.widget_config.stations?.token} requestedTime={requestedTime} />;
   }
-  return <Unavailable />;
+
+  return process.env.EXPO_PUBLIC_ENABLE_WEATHER_MAP ? (
```

Comment:
I've been adding new environment variables to `.env.sample` so that they're somewhat documented in one place. can you add `EXPO_PUBLIC_ENABLE_WEATHER_MAP` to that file?

https://github.com/NWACus/avy/blob/4b17f4b44ccc7d92cd3f50cdb524d0129cd1041a/.env.sample#L34-L38